src/picongpu/include/particles/particleToGrid/derivedAttributes/LarmorPower.hpp

```diff
@@ -55,16 +55,19 @@ namespace derivedAttributes
         const float_X gamma2 = gamma * gamma;
         const float_X c2 = SPEED_OF_LIGHT * SPEED_OF_LIGHT;
 
-        const float3_X mom_dt = mom - mom_mt1;
+        const float3_X mom_dt = (mom - mom_mt1) / float_X(DELTA_T);
         const float_X el_factor = charge * charge
             / (float_X(6.0) * PI * EPS0 *
-               c2 * c2 * SPEED_OF_LIGHT * mass * mass);
-        const float_X energyLarmor = el_factor * gamma2 * gamma2
-            * (math::abs2(mom_dt) -
-               math::abs2(math::cross(mom, mom_dt)));
+               c2 * SPEED_OF_LIGHT * mass * mass) * gamma2 * gamma2;
```

Comment:
in #1262 we thought it should be gamma^4 so right now `gamma2 * gamma2` is that implementation. is the description in #1262 wrong?